agent/VERSION

```diff
@@ -0,0 +1 @@
+3.39.0
```

Comment:
i got a little bit confused by this initially, thinking "shouldn't this be 3.39.1"?, but then i realised that 3.39.1 is on another branch, that isn't up do date with master. non-linear development is confusing sometimes.

Reply:
I initially did it as 3.39.1 before realising I was targeting `main` :D